src/Abstractions/NexusMods.Abstractions.Games.FileHashes/IFileHashesService.cs

```diff
@@ -0,0 +1,19 @@
+using NexusMods.MnemonicDB.Abstractions;
+
+namespace NexusMods.Abstractions.Games.FileHashes;
+
+/// <summary>
+/// Interface for the file hashes service, which provides a way to download and update the file hashes database
+/// </summary>
+public interface IFileHashesService
+{
+    /// <summary>
+    /// Force an update of the file hashes database
+    /// </summary>
+    public Task ForceUpdate();
+    
+    /// <summary>
+    /// Get the file hashes database, downloading it if necessary
+    /// </summary>
+    public ValueTask<IDb> GetFileHashesDb();
```

Comment:
So the idea is to have a single DB that contains all the hashes from all the versions of all the supported games?

Would it make sense in the future to potentially split up this DB based on games and only fetch the relevant DBs when a user starts managing a game?
And likewise, only query for updates to the DBs for which the user has games being managed?

Reply:
Yes for now, a single DB, it's just easier to maintain for now. It's easy enough in the future to split it up if it gets too large.